batch-setup/make_rawr_tiles.py

```diff
@@ -219,6 +239,12 @@ def make_rawr_tiles(rawr_config_file, missing_config_file, missing_bucket,
                         "written out by make_tiles.py")
     parser.add_argument('missing_bucket', help="Bucket to store tile "
                         "enumeration logs in while calculating missing tiles.")
+    parser.add_argument('--use-tiles-coords-generator', type=lambda x: bool(strtobool(x)), nargs='?',
+                        const=True, default=False)
+    parser.add_argument('--coords-generator-bbox-west', type=float, help="west longitude of bounding box to build")
+    parser.add_argument('--coords-generator-bbox-south', type=float, help="south latitude of bounding box to build")
+    parser.add_argument('--coords-generator-bbox-east', type=float, help="east longitude of bounding box to build")
+    parser.add_argument('--coords-generator-bbox-north', type=float, help="north latitude of bounding box to build")
```

Comment:
sorry I wrote this comment yesterday but didn't submit for some reason -  I think these should all be one arg.
`--coords-generator-bbox=1,2,3,4`
you can just split on commas and assert 4 floats or whatevs.  
https://wiki.openstreetmap.org/wiki/Bounding_Box

Reply:
See what we do in Tilequeue here:

- https://github.com/tilezen/tilequeue/blob/33a4dd42e8f764bcc9817e89554ee21922f501a8/tilequeue/config.py#L62-L72

Reply:
changed in the newest commit